onnxscript/function_libs/torch_aten/graph_building.py

```diff
@@ -306,55 +403,21 @@ def add_function(
         # Compute outputs from the function schema
 
         result = self._add_torchscript_op(
-            f"{onnx_function.function_ir.domain}::{onnx_function.name}", onnx_inputs, onnx_attributes, outputs=len(onnx_function.function_ir.outputs)
+            f"{onnx_function.function_ir.domain}::{onnx_function.name}",
+            onnx_inputs,
+            onnx_attributes,
+            n_outputs=len(onnx_function.function_ir.outputs),
         )
 
         return result
 
-    def _wrap_constant_to_torchscript_value(self, constant) -> torch.Value:
-        if isinstance(constant, float):
-            # Always promote scalar to tensor with element type "dtype."
-            # Usually, "dtype" is extracted from the expected output tensor of the node.
-            # If this assumption is broken, we probably need to
-            #  1. add "scalar" type in ONNX  and extend all exporters to support it, or
-            #  2. write type promotion logic for each operator.
-            # TODO(wechi): the called exporting function should tell all allowed input and output types.
-            # Then, here we can try type-casting if type-mismatch happens.
-            value = self.graph.op(
```

Comment:
I wonder how we passed this? Isn't torch._C.graph.op() deprecated?

Reply:
It was deprecated but still functional (not removed)